src/languages/php.js

```diff
@@ -394,6 +393,42 @@ export default function(hljs) {
     ]
   };
 
+  const FUNCTION_INVOKE = {
+    relevance: 0,
+    match: [
+      /\b/,
+      // to prevent keywords from being confused as the function title
+      regex.concat("(?!fn\\b|function\\b|", normalizeKeywords(KWS).join("\\b|"), "|", normalizeKeywords(BUILT_INS).join("\\b|"), "\\b)"),
+      IDENT_RE,
+      regex.concat(WHITESPACE, "*"),
+      regex.lookahead(/(?=\()/)
+    ],
+    scope: {
+      3: "title.function.invoke",
+    }
+  };
+  FUNCTION_INVOKE.contains = [
```

Comment:
```suggestion
    contains:  [
```

Reply:
or some such.

Reply:
TS2448: Block-scoped variable 'FUNCTION_INVOKE' used before its declaration.

Reply:
Oh.  Define what you can, only do the modification after.

```
PARAMS_MODE.contains.push(FUNCTION_INVOKE);
```